_harp/about/_company.md

```diff
@@ -1,18 +1,18 @@
 # About us
 
-Founders & Coders runs a well-established coding camp in London. We are delighted to announce that in 2017 we will be opening our first international campus in the Arab Israeli city of Nazareth.
+Founders & Coders runs a well-established fullstack JavaScript coding camp in London. We are delighted to announce that in 2017 we will be opening our first international campus in the Arab Israeli city of Nazareth.
 
-We follow a highly-effective peer-led approach to learning and we do not employ any teaching or administrative staff. Our low cost of operation allows us to offer our training without charging any fees to our students.
+We follow a highly-effective peer-led approach to learning and we do not employ any teaching or administrative staff. Thanks to the dedication and effort of our two directors, we are currently able to offer our training without charging any fees to our students.
```

Comment:
As I mentioned, I really didn't mean this - when I said 'we' I was referring to all of us who have put time into FAC!